### PR TITLE
AVT-1141: Review and fix : AVT-1141,AVT-1142,AVT-1144

### PR DESCRIPTION
**Pull request by tranducanh**
## Description
Review and fix : AVT-1141,AVT-1142,AVT-1144
(https://perzoinc.atlassian.net/browse/AVT-1141)
(https://perzoinc.atlassian.net/browse/AVT-1142)
(https://perzoinc.atlassian.net/browse/AVT-1144)

## Run
Please refer attachment for more details
[Spectron Tests Report.zip](https://github.com/symphonyoss/SymphonyElectron/files/2359386/Spectron.Tests.Report.zip)

## QA Checklist
- [] Unit-Tests
- [x] Automation-Tests

### CODE REVIEW

File: tests/spectron/spectronWindowsActions.js [hunk 1]
```diff
@@ -145,9 +157,46 @@ class WindowsActions {
         });
     }
 
+    async actionForMenusOnMac(arrMenu) {
+        let webAction = await new WebActions(this.app);
+        //await this.app.browserWindow.getBounds().then(async (bounds) => {
```
Comment (KiranNiranjan) [inline tests/spectron/spectronWindowsActions.js]: Remove commented code

File: tests/spectron/spectronWindowsActions.js [hunk 2]
```diff
@@ -145,9 +157,46 @@ class WindowsActions {
...
+            await robot.setMouseDelay(2000);
+            let x = 5;
+            let y = 5;
+            await robot.moveMouseSmooth(x, y);
+            await robot.moveMouse(x, y);
+            await robot.mouseClick();
+            //await webAction.openApplicationMenuByClick();
```
Comment (KiranNiranjan) [inline tests/spectron/spectronWindowsActions.js]: Same here

File: tests/spectron/spectronWindowsActions.js [hunk 3]
```diff
@@ -145,9 +157,46 @@ class WindowsActions {
...
+                        await robot.keyTap('down');
+                    }
+                }
+                if (arrMenu.length > 1 && i != arrMenu.length - 1) {
+                    //handle right keygen
+                    await robot.keyTap('right');
+                }
+            }
+            await robot.keyTap('enter');
+        //});
```
Comment (KiranNiranjan) [inline tests/spectron/spectronWindowsActions.js]: Same here

File: tests/spectron/bringToFront.spectron.js [hunk 4]
```diff
@@ -1,118 +1,39 @@
...
-            }
-
-            if (isWindowsOS) {
-                expect(results[2]).toBe(true);
-            }
-            done();
-        }).catch((err) => {
-            done.fail(new Error(`bringToFront failed to focus with error: ${err}`));
-        });
+    it('should restore the browser window and verify window focus', async (done) => {
```
[the remark this answers is not in the document]
Comment (tranducanh) [reply tests/spectron/bringToFront.spectron.js]: fixed

**Comment by tranducanh** [comment]
My result
[Spectron Tests Report.zip](https://github.com/symphonyoss/SymphonyElectron/files/2359551/Spectron.Tests.Report.zip)

File: tests/spectron/spectronWindowsActions.js (hunk 3 above)
Comment (tranducanh) [reply tests/spectron/spectronWindowsActions.js]: fixed

File: tests/spectron/spectronWindowsActions.js (hunk 2 above)
Comment (tranducanh) [reply tests/spectron/spectronWindowsActions.js]: fixed

File: tests/spectron/spectronWindowsActions.js (hunk 1 above)
Comment (tranducanh) [reply tests/spectron/spectronWindowsActions.js]: fixed

**Comment by KiranNiranjan** [comment]
@tranducanh Please resolve conflicts

**Pull request merged**